include/swift/RemoteInspection/BitMask.h

```diff
@@ -74,7 +74,16 @@ class BitMask {
 
   BitMask(unsigned sizeInBytes, uint64_t sourceMask): size(sizeInBytes) {
     mask = (uint8_t *)calloc(1, sizeInBytes);
-    memcpy(mask, &sourceMask, sizeInBytes);
+    if (!mask) {
```

Comment:
This is just here to further harden the bit mask code against allocation failures.  Allocation failures can occur in this code when clients try to use RemoteMirror to decipher type data from random chunks of memory.

Reply:
Should we also have a limit on `sizeInBytes` so we don't read garbage and then successfully, but unhappily, allocate 500TB of memory?